core/modules/color/color.module

```diff
@@ -186,13 +185,19 @@ function color_css_alter(&$css) {
           'map' => array(),
         );
         foreach ($theme_info['css'] as $file) {
-          $stylesheet = file_get_contents($theme_path . '/' . $file);
-          _backdrop_build_css_path(NULL, base_path() . dirname($paths['source'] . $file) . '/');
-          $stylesheet = preg_replace_callback('/url\([\'"]?(?![a-z]+:|\/+)([^\'")]+)[\'"]?\)/i', '_backdrop_build_css_path', $stylesheet);
+          $filename = _color_get_file_path($file, $preview_theme);
+          $stylesheet = file_get_contents($filename);
+          $paths['source'] = backdrop_get_path('theme', _color_get_theme_from_file($file, $preview_theme));
+          $base = base_path() . dirname($filename) . '/';
+          _backdrop_build_css_path(NULL, $base);
+
+          $paths['source'] = backdrop_get_path('theme', _color_get_theme_from_file($file, $preview_theme));
```

Comment:
This line is repeated twice. Here and on line 190. Mistake?

Reply:
Ahh, yes. I added a new commit that removes the extra line.